docs/mkdocs-base.yml

```diff
@@ -13,7 +13,7 @@ copyright: Kartoza and Contributors
 
 markdown_extensions:
         - attr_list # lets us resize images see https://github.com/mkdocs/mkdocs/issues/1678#issuecomment-455500757
-        - sane_lists # helps make numbering in lists correct
+        - sane_lists # lets us resize images see https://github.com/mkdocs/mkdocs/issues/1678#issuecomment-455500757
```

Comment:
Please don't change the comment here. The current `# helps make numbering in lists correct` is the correct in line comment